crowdin/messages.json

```diff
@@ -88,7 +88,6 @@
   "122171127": "Trade multipliers on synthetic indices 24/7, 365 days a year. Trade multipliers on forex round the clock on weekdays.",
   "123005346": "No artificial barriers to customer withdrawals",
   "123291381": "What is the rebranding about?",
-  "125131030": "Thanks for subscribing to Deriv Academy",
```

Comment:
Hi @mohammad-hashemi-deriv can we untrack this message.json file from this PR, otherwise we won't be able to revert this in the future if needed.